src/tfr_msgs/msg/SystemStatus.msg

```diff
@@ -0,0 +1,3 @@
+time SendTime
```

Comment:
So instead of sending a timestamp like this I am going to suggest we use the standard ros header object for consistency. Pretty much every top level msg data type in ros has one. 

A concrete benefit this gives us is that ros will automagically assign each message sent like this a unique seq number, and there is greater interoperability with existing messsage apis, if that ever becomes a priority.

It's described pretty well in this article in the header section. It's just a convention thing.
http://wiki.ros.org/msg

Reply:
@Jebbs Disregard this! I just realized this is only applicable for things we want to integrate with tf, which we would never have reason to with this. so if you just want a timestamp for internal stuff, this is more appropriate.

I did some more reading and changed my mind.